app/src/main/java/io/seqera/wavelit/App.java

```diff
@@ -79,10 +80,12 @@ public class App implements Runnable {
     @Option(names = {"--context"}, description = "Directory path where the build context is stored.")
     private String contextDir;
 
-
     @Option(names = {"--layer"})
     private List<String> layerDirs;
 
+    @Option(names = {"--config-entrypoint"}, description = "Entrypoint for the container image name to be provisioned.")
+    private List<String> entryPoint;
```

Comment:
```suggestion
    private List<String> entrypoint;
```

I know it's arguable, however Docker convention is lowercase

Reply:
done